docs/user/configuration-management.rst

```diff
@@ -191,82 +189,93 @@ Change IntelMQ process manager in the *global* configuration:
 
 .. code-block:: json
 
-    "deduplicator-expert": {
-        "source-queue": "deduplicator-expert-queue",
-        "destination-queues": [
-            "taxonomy-expert-queue"
-        ]
-    }
+   {
+       "malware-domain-list-collector": {
+           "group": "Collector",
+           "name": "Malware Domain List",
+           "module": "intelmq.bots.collectors.http.collector_http",
+           "description": "Malware Domain List Collector is the bot responsible to get the report from source of information.",
+           "enabled": false,
+           "parameters": {
+               "http_url": "http://www.malwaredomainlist.com/updatescsv.php",
+               "feed": "Malware Domain List",
+               "rate_limit": 3600
+           }
+       }
+   }
 
-And a bot with two destination queues:
+Pipeline Configuration
+======================
 
-.. code-block:: json
+The pipeline configuration defines how the data is exchanges between the bots. For each bot, it defines the source queue (there is always only one) and one or multiple destination queues. This section shows the possibilities and definition as well as examples. The configuration of the pipeline can be done by the |intelmq-manager-github-link|  with no need to intervene manually. It is recommended to use this tool as it guarantees that the configuration is correct. The configuration of the pipelines is done in the ``runtime.conf`` as part of the individiual bots settings.
 
-    "cymru-whois-expert": {
-        "source-queue": "cymru-whois-expert-queue",
-        "destination-queues": [
-            "file-output-queue",
-            "misp-output-queue"
-        ]
-    }
+Source queue
+------------
 
-These are the usual configurations you mostly see.
+This setting is **optional**, by default, the source queue is the bot ID plus "-queue" appended.
+For example, if the bot ID is ``example-bot``, the source queue name is ``example-bot-queue``.
 
-Named queues / paths
---------------------
+.. code-block::
 
-Beginning with version 1.1.0, queues can be "named", these are the so-called *paths*. The following two configurations are equivalent:
+   "source-queue": "example-bot-queue"
 
-.. code-block:: json
+For collectors, this field does not exist, as the fetch the data from outside the IntelMQ system by definition.
 
-   "destination-queues": ["taxonomy-expert-queue"]
-   "destination-queues": {"_default": ["taxonomy-expert-queue"]}
+Destination queues
+------------------
 
-As we can see the *default* path name is obviously ``_default``. Let's have a look at a more complex and complete example:
+Destination queues are defined using a dictionary with a custom name as key and a list of queue-identifiers as the value.
 
 .. code-block:: json
 
```

Comment:
In the example below, which I can't comment on directly in this UI, there's still the old and unsupported syntax with strings as value of the dict (default and error paths) instead of strings in lists (other path).